src/Symfony/Component/Inflector/phpunit.xml.dist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
```

Comment:
Every other component uses `4.1` though. Perhaps this should be addressed in a separate PR (if it's an issue)?